types/index.d.ts

```diff
@@ -526,4 +526,464 @@ declare module '@azure/functions' {
             lastUpdated: string;
         };
     }
+
+    /**
```

Comment:
Wooooaahhhh I just meant a simple internal interface haha. This file is considered "external" because users can access these type. Anything _not_ exported in this file is "internal"

And for our purposes, we only need
```
export interface PackageJson {
    type?: whatever
}
```